builtins/generic.ispc

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2024, Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+// This file provides the target-independent definitions for builtin functions.
+
+// Empty for now.
+void foo() {}
```

Comment:
Do we want that popular function name here? Would it clash with user `foo()`? As an alternative, I would suggest just `;`, which is a valid empty file.

Reply:
Right, it works fine with `;`.